### PR TITLE
Rev concl jm

### PR DESCRIPTION
**Pull request by jannes-m**
Hey Robin, 
here, you find some suggestions for the last chapter. One thing, is it necessary that we call it "social benefit"? Can we broaden it a bit? Geocomputation for the common good/for everybody? I mean the applications don't have to have a social touch (though they can). I mean it's really nice that you can use the tools if they are needed, for example in emergency cases/crisis areas, etc.. Makes me think of OSM's https://tasks.hotosm.org/. This is nice but I think it is just one application, therefore, I would broaden the term a bit while one still can point out that it would be nice to use geocomputation for the social benefit. But then, we have to prepare ourselves for questions like, when have you used geocomputation the last time for saving/helping the world/society/poor? Have you?

### CODE REVIEW

File: 15-synthesis.Rmd [hunk 1]
```diff
@@ -55,10 +55,11 @@ The range of options grows further when other R packages are considered: you cou
 We recommend using **sf** and the other packages showcased in this book, for reasons outlined in Chapter \@ref(spatial-class), but it's worth being aware of alternatives and being able to justify your choice of software.
 
 We deliberately covered both **tidyverse** and base R approaches to attribute data operations.
-Chapter \@ref(attr) showed how both `nz["Name"]` and `nz %>% select(Name)` can be used to achieve the same result, for example.
+Chapter \@ref(attr) showed how both `nz[, "Name"]` and `nz %>% select(Name)` can be used to achieve the same result, for example.
```
Comment (Nowosad) [inline 15-synthesis.Rmd]: @jannes-m The previous version was probably better. 
- On sf object, both `nz["Name]` and `nz[, "Name"]` give the same result
- However on data.frame objects `nz_df["Name"]` is the same as `nz_df %>% select(Name)`
- `nz_df[, "Name"]` is more like `nz_df %>% pull(Name)`

Comment (jannes-m) [reply 15-synthesis.Rmd]: Robin, while on the bike today,what do you think about replacing "Geocomputation for social benefit" by "Advocating open source software"? This is broader, we all agree, and you can still say that using an open source approach, social benefits are more likely to happen.

Comment (Nowosad) [reply 15-synthesis.Rmd]: @jannes-m `nz["Name"]` and `nz[, "Name"]` give the same output - an sf object with two columns. This is due to "stickiness" of a geometry column.

**Comment by Robinlovelace** [comment]
Will aim to look at this this evening.

**Comment by jannes-m** [comment]
@Nowosad yeah, I got this. I was talking of the `data.frame` case.

**Comment by jannes-m** [comment]
:+1:

**Pull request merged**